incuna-sass/mixins/_svg.sass

```diff
@@ -0,0 +1,20 @@
+//////////  S V G   M I X I N  //////////////////////////////
+// To use like you would a sprite with the added benefit that
+// it can be whatever size you like and will work perfectly
+// on retina screens
+
+// Element mixin to be used for general background images
+// as you would use a retina mixin
+@mixin svg-element($file, $width: 100%, $height: $width)
+    @include background(image-url('svg/#{$file}.svg'))
```

Comment:
ah that's a good point actually, we're going to try to get rid of compass stuff anyway